Documentation/getting-started.md

```diff
@@ -0,0 +1,69 @@
+# Getting started
+
+## Before starting
+
+The Adobe Experience Platform Consent for Edge Network extension has the following peer dependency, which must be installed prior to installing the consent extension:
+- [Mobile Core](https://developer.adobe.com/client-sdks/documentation/mobile-core)
+
+## Configure the Adobe Experience Platform Consent for Edge Network extension in Data Collection UI
+1. Log into [Adobe Experience Platform Data Collection](https://experience.adobe.com/data-collection).
+2. From **Tags**, locate or search for your Tag mobile property.
+3. In your mobile property, select **Extensions** tab.
+4. On the **Catalog** tab, locate or search for the **Consent** extension, and select **Install**.
+5. Set your desired default consent level.
+6. Select **Save**.
+7. Follow the [publishing process](https://developer.adobe.com/client-sdks/documentation/getting-started/create-a-mobile-property/#publish-the-configuration) to update SDK configuration.
+
+> **Note**
+> In order to ingest and use the data collected by this extension, follow the guide on [ingesting data using the Consents and Preferences data type](https://experienceleague.adobe.com/docs/experience-platform/xdm/data-types/consents.html#ingest).
+
+> **Warning**
+> The use of this extension is currently limited to the setting (and enforcement) of client-side, macro consent flags. While SDK APIs allow for granular and global consent preference collection, flags are not consistently enforced with upstream applications and therefore will not accommodate use cases that rely on global/granular consent preferences.
+
+## Add the AEP Consent extension to your app
+
+### Download and import the Consent extension
+
+1. Add the Mobile Core and Edge extensions to your project using the app's Gradle file.
+
+  ```java
+implementation 'com.adobe.marketing.mobile:core:1.+'
+implementation 'com.adobe.marketing.mobile:edge:1.+'
+implementation 'com.adobe.marketing.mobile:edgeidentity:1.+'
+implementation 'com.adobe.marketing.mobile:edgeconsent:1.+'
+  ```
+
+2. Import the Mobile Core and Edge extensions in your Application class.
+
+  ```java
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Edge;
+import com.adobe.marketing.mobile.edge.identity.Identity;
+import com.adobe.marketing.mobile.edge.consent.Consent;
+  ```
+
+3. Register Edge extensions with Mobile Core
+
+  ```java
+public class MobileApp extends Application {
+
+    @Override
+    public void onCreate() {
+      super.onCreate();
+      MobileCore.setApplication(this);
+      try {
+        Edge.registerExtension();
+        Consent.registerExtension(); // register Consent
+        Identity.registerExtension();
+        // register other extensions
+        MobileCore.start(new AdobeCallback() {
```

Comment:
Can you update this example for 2.0 by using the new `MobileCore.registerExtensions(List, AdobeCallback)` API?